src/bls12_381/ec.rs

```diff
@@ -1419,6 +1465,39 @@ pub mod g2 {
             }
         }
 
+        fn get_swenc_const0() -> Fq2 {
+            Fq2 {
+                c0: super::super::fq::SWENC_CONST0,
+                c1: Fq::zero()
+            }
+        }
+
+        fn get_swenc_const1() -> Fq2 {
+            Fq2 {
+                c0: super::super::fq::SWENC_CONST1,
+                c1: Fq::zero()
+            }
+        }
+
+        /// Maps any elliptic curve point to the subgroup G2,
```

Comment:
Well, it cannot map points from small subgroups into the G2 subgroup. I would rather call this `scale_by_cofactor`.